installer/templates/phx_web/views/helpers.ex

```diff
@@ -1,10 +1,79 @@
-defmodule <%= @web_namespace %>.ErrorHelpers do
+defmodule <%= @web_namespace %>.Helpers do
   @moduledoc """
   Conveniences for translating and building error messages.
   """<%= if @html do %>
 
   use Phoenix.HTML
 
+  import Phoenix.LiveView
+  import Phoenix.LiveView.Helpers
```

Comment:
You can simplify this a bit with `use Phoenix.Component`:

```suggestion
  use Phoenix.Component
```